tla-types/src/main/scala/at/forsyte/apalache/tla/types/DependencyGraph.scala

```diff
@@ -150,13 +152,20 @@ object DependencyGraph {
         _ ++ _
       }
     case LetInEx( letInBody, defs@_* ) =>
-      ( defs map { case TlaOperDecl( _, _, b ) => subCalls( b ) } ).foldLeft(
+      ( defs map {
+        case TlaOperDecl( _, params, b ) =>
```

Comment:
I think a small helper function to self-document what's being done here could be helpful here. From the perspective of an unfamiliar reader (this one, any how :) ), the level of tested operations here is pretty hard to parse out.